data_collection/gazette/spiders/ba/ba_rio_real.py

```diff
@@ -0,0 +1,15 @@
+from datetime import date
+
+from gazette.spiders.base.brtransparencia import BaseBrTransparenciaSpider
+
+
+class BaRioRealSpider(BaseBrTransparenciaSpider):
+    name = "ba_rio_real"
+    TERRITORY_ID = "2927002"
+    allowed_domains = [
+        "cmriorealba.brtransparencia.com.br",
+        "api.brtransparencia.com.br",
+    ]
+    start_urls = ["https://http://cmriorealba.brtransparencia.com.br/diario.html"]
```

Comment:
@talesmota acredito que o link aqui seja apenas http:
```suggestion
    start_urls = ["http://cmriorealba.brtransparencia.com.br/diario.html"]
```